### DISCUSSION TITLE
Support alternate (lower fee) payout options

### DISCUSSION THREAD

**Discussion by alromh87** [418sec/huntr#884] [Ideas]
I noticed that PayPal fee (5%) is high compared to other systems like TransferWise (less than 1%)

It would be nice if lower fee option would be added.

**Comment by JamieSlome** [comment]
@alromh87 - thanks for raising this. Would it be possible to turn this into a Discussion?

I will get some feedback from the team and share it with you there...

Cheers! 🍰

**Comment by adam-nygate** [comment]
How do people feel regarding Bitcoin?

**Reply by JamieSlome** [reply]
Ooo... 💰

**Reply by TheCrott** [reply]
Agree with crypto support

**Comment by mzfr** [comment]
Another lower payment option at least for all the Indians would be [razorpay/upi](https://razorpay.com/upi/).

**Reply by arjunshibu** [reply]
Yeah I agree with that. The fee PayPal charged me for the last payment was about 120$😢

**Reply by Anon-Artist** [reply]
Yeah i too support the statement from @mzfr i had lossed nearly 40$ from my previous payment 😐.
I dont know about transferwise if it is good go for it upi is good for indians not sure about other countries.

**Reply by Asjidkalam** [reply]
+1
Razorpay would be a good option for Indians.

**Reply by b1nslashsh** [reply]
+1

**Reply by d3m0n-r00t** [reply]
+1 
Razorpay would be good for huntrs in India.

**Reply by TheCrott** [reply]
Is it available only for Indians?

**Comment by TheCrott** [comment]
Add payoneer support, it's popular but I don't know the fee

**Comment by B3EF** [comment]
It will be great too change the payment method since the fee taken by paypal is somewhat a greater amount .

**Comment by B3EF** [comment]
> How do people feel regarding Bitcoin?

There is no use for bitcoin here in India , I think so

**Reply by TheCrott** [reply]
But you can withdraw bitcoin in local crypto exchange and convert it to rupee

**Reply by B3EF** [reply]
then i think so we will also have to pay a conversion fee for that too .

**Reply by TheCrott** [reply]
Sure it does

**Comment by b1nslashsh** [comment]
As @alromh87 said , support for TransferWise is acceptable 👍

**Reply by TheCrott** [reply]
+1

After searching for reviews, transferwise is a good option. It's fast and have generous fee. Here are the links

[Transferwise vs Payoneer with transfer case study](https://moneytransfers.com/comparison/transferwise-vs-payoneer)
[Transferwise Business Account](https://transferwise.com/us/blog/payoneer-business-account-alternative)
[Transferwise vs Payoneer](https://www.bloggersideas.com/transferwise-vs-payoneer/)

**Comment by AkashHamal0x01-zz** [comment]
the best option is crypto, the huntr platform should pay in coinbase mail address of users, it has two advantages on both sides,

- no transaction fee! saving on both ends
- instant transfers!

**Reply by b1nslashsh** [reply]
Then as @B3EF said, " we will also have to pay a conversion fee for that too ."
